<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hilvl_coorGen</class>
 <widget class="QWidget" name="hilvl_coorGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tab_coorGenMod">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>641</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_coorGen">
    <attribute name="title">
     <string>Basic</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="combo_coorGen"/>
     </item>
     <item>
      <widget class="QWidget" name="wdg_coorGen_holder" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_coorMod">
    <attribute name="title">
     <string>Advanced (modifier)</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="combo_coorMod"/>
     </item>
     <item>
      <widget class="QWidget" name="wdg_coorMod_holder" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="wdg_coorHub_holder" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="wdg_3Dmod_holder" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
